docs/faq/data-loading.md

```diff
@@ -2,20 +2,29 @@
 
 ## **Why don’t I see any data in my destination yet?**
 
-It can take a while for Airbyte to load data into your destination. Some sources have restrictive API limits which constrain how much data we can sync in a given time. Large amounts of data in your source can also make the initial sync take longer. You can check your sync status in your connection detail page that you can access through the destination detail page or the source one.
+It can take a while for Airbyte to load data into your destination. Some sources have restrictive API limits which constrain how much 
+data we can sync in a given time. Large amounts of data in your source can also make the initial sync take longer. You can check your
+sync status in your connection detail page that you can access through the destination detail page or the source one.
 
 ## **What happens if a sync fails?**
 
 You won't loose data when a sync fails, however, no data will be added or updated in your destination.
 
-Airbyte will automatically attempt to replicate data 3 times. You can see and export the logs for those attempts in the connection detail page. You can access this page through the Source or Destination detail page.
+Airbyte will automatically attempt to replicate data 3 times. You can see and export the logs for those attempts in the connection 
+detail page. You can access this page through the Source or Destination detail page.
 
-In the future, you will be able to configure a notification \(email, Slack...\) when a sync fails, with an option to create a GitHub issue with the logs. We’re still working on it, and the purpose would be to help the community and the Airbyte team fix the issue as soon as possible, especially if it is a connector issue.
+In the future, you will be able to configure a notification \(email, Slack...\) when a sync fails, with an option to create a 
```

Comment:
This is actually live

Reply:
Thanks @michel-tricot, also I created https://github.com/airbytehq/airbyte/issues/3976 to made a small doc/tutorial to bring more visibility about this feature.